src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueTest.java

```diff
@@ -33,19 +33,51 @@ void setUp() {
     }
 
     @Test
-    void whenMeshInboundQueuePathologyMessageIsReceivedThenMessageIsHandled()
+    void whenMeshInboundQueuePathology2MessageIsReceivedThenMessageIsHandled()
             throws IOException, JMSException, JSONException {
         final MeshMessage meshMessage = new MeshMessage()
-            .setWorkflowId(WorkflowId.PATHOLOGY)
-            .setContent(new String(Files.readAllBytes(getPathologyEdifactResource().getFile().toPath())))
+            .setWorkflowId(WorkflowId.PATHOLOGY_3)
+            .setContent(new String(Files.readAllBytes(getPathology2EdifactResource().getFile().toPath())))
```

Comment:
Should this be WorkflowId.PATHOLOGY_2 ?
```suggestion
            .setWorkflowId(WorkflowId.PATHOLOGY_2)
            .setContent(new String(Files.readAllBytes(getPathology2EdifactResource().getFile().toPath())))
```

Reply:
Thanks - tests still pass because we haven't actually changed anything yet